kubelet-to-gcm/monitor/kubelet/translate.go

```diff
@@ -602,12 +558,14 @@ func (t *Translator) getMonitoredResource(labels map[string]string) *v3.Monitore
 		}
 	}
 
-	resourceLabels["location"] = t.zone
+	resourceLabels["location"] = t.clusterLocation
 
-	if _, found := labels["node"]; found {
-		resourceLabels["node_name"] = labels["node"]
+	if _, found := labels["container"]; !found {
+		if t.instanceID != "" {
+			resourceLabels["node_name"] = t.instanceID
```

Comment:
This should be the node name, not numeric instance id.

Reply:
Changed from "instanceID" to "instance".

Reply:
This shouldn't be changed, both should be available. "node_name" should use instance name, but "instance_id" label should use the instance id, same as before.

Reply:
"instance_id" label for "gke_container" schema was using instance name before, and that remains unchanged.